From 4c0954f77ba05d77192ee1519929a39fbc978321 Mon Sep 17 00:00:00 2001
From: Alex Kiernan <alex.kiernan@gmail.com>
Date: Mon, 5 Dec 2022 15:14:22 +0000
Subject: [PATCH 3/6] mDNSShared: Drop MacOS specific __block qualifier

Support for this extension only exists in MacOS/Clang, also it's not
actually used here, so we can just drop it.

Upstream-Status: Pending
Signed-off-by: Alex Kiernan <alex.kiernan@gmail.com>
---
 mDNSShared/uds_daemon.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mDNSShared/uds_daemon.c b/mDNSShared/uds_daemon.c
index 9ae5f78542d6..8c006b71a4ea 100644
--- a/mDNSShared/uds_daemon.c
+++ b/mDNSShared/uds_daemon.c
@@ -2912,7 +2912,7 @@ exit:
 mDNSlocal mStatus add_domain_to_browser(request_state *info, const domainname *d)
 {
     browser_t *b, *p;
-    __block mStatus err;
+    mStatus err;
 
     for (p = info->u.browser.browsers; p; p = p->next)
     {
-- 
2.35.1

